prow/config/branch_protection_test.go

```diff
@@ -0,0 +1,408 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"reflect"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+func TestJobRequirements(t *testing.T) {
```

Comment:
Consider dropping the duplicated prow/cmd/branchprotector test cases with a `TODO(fejta): move them in a later PR` type comment (or do this move in a separate commit)

Reply:
I did 3 commits PTAL